src/main/kotlin/calculator/view/InputView.kt

```diff
@@ -0,0 +1,12 @@
+package calculator.view
+
+class InputView {
```

Comment:
static 한 값과 메소드만을 가지고 있는 클래스로 보이네요~
해당 클래스를 싱글톤 객체인 `object` 로 만들어보는 것은 어떨까요?
```suggestion
object InputView {
```

Reply:
`InputView`, `ResultView` 클래스는 싱글톤으로 생성되게끔 하는게 적합해 보입니다! 😃 
`InputView`, `ResultView` 클래스에 싱글턴을 위한 `object` 키워드 적용하였습니다!

- https://github.com/next-step/kotlin-lotto/commit/c7c5928159b867e0431d2782dae1ab550dd94aec
- https://github.com/next-step/kotlin-lotto/commit/084314ca30ab421aa32a6e5a5fa2d21fb501412b